documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -0,0 +1,430 @@
+*In the case of a operational environment certification, the scope and area of application in particular must be stated.*
+
+### §3.2 The applicant\'s obligations to cooperate      
+1. The applicant shall carry out the cooperation activities required for proper evaluation and certification or, if applicable, those contractually agreed at his own expense.
+2. In particular, the applicant shall be obliged to provide the Evaluation Facility with or allow inspection of sufficient documentation on the evaluation item.
+  * a. In the case of operational environment certification, the documentation shall include a role or service description as well as the technical and organizational measures taken by the applicant in accordance with the IDS criteria catalog.
+  * b. In the case of component certification, the object of the manufacturer documentation is, in addition to an architectural description, in particular the source code and the organizational measures for secure software development in accordance with the IDS criteria catalog.
+3. The applicant assures the evaluation facility and certification body that the measures specified in the documentation have been fully implemented.
+4. Where the applicant seeks recognition of existing certificates for his component or organization, he shall communicate the request for recognition before the start of the evaluation, specifying the exact designation of the certificate to be recognized and the component for which recognition is sought, and submit the documents relevant for the assessment of recognition.
+
+### §3.3 Procedure of the evaluation     
+1. The evaluation shall be based on the clearly defined description of the evaluation item in the contract and shall include at least one evaluation of the documentation provided by the applicant (para. 2), inquires and interviews (para. 3) and an on-site evaluation (para. 4). In the case of component certification, technical tests (para. 5) shall be carried out.
+2. With the documentary assessment, the evaluator checks compliance with the requirements of the IDS on the basis of the information in the applicant\'s documentation.
+3. Interviewing the applicant\'s staff or other persons involved in the provisioning of the evaluation item may be used to determine the facts of individual aspects and to verify the accuracy of the documentation.
+In particular, it shall be used to verify aspects identified as critical by the evaluator. Inquiries can be conducted in writing or in person.
+In any case, those concerning central aspects should be conducted as oral interviews.
+Where a personal interview would be disproportionate, it may take the form of videoconferencing.
+4. The on-premise evaluation shall include at least an evaluation of the procedures on the premises of the applicant or its subcontractors.
+5. In the case of component certification, appropriate functional and security tests are conducted.
```

Comment:
```suggestion
5. In the case of component certification, appropriate functional and security tests, including penetration testing, are conducted.
```

Reply:
For the current revision, I would keep it as vague here as it is currently defined since we should discuss with the WG whether we want to add further details including mentioning of Source Code Audits and differentiating the Assurance Levels...